src/components/admin/donations/grid/RenderEditPersonCell.tsx

```diff
@@ -86,12 +87,18 @@ export default function RenderEditPersonCell({
           </Box>
         )}
         isOptionEqualToValue={(option, value) => option.firstName === value.firstName}
-        filterOptions={createFilterOptions<PersonResponse>({
-          matchFrom: 'any',
-          limit: 5,
-          ignoreCase: true,
-          trim: true,
-        })}
+        filterOptions={(options, state) => {
+          const displayOptions = options.filter(
+            (option) =>
+              option.firstName
```

Comment:
can we extract this to a function like:

```
function contains(value: string, pattern: string): boolean {
  const valueCleaned: string = value.toLowerCase().trim();
  const patternCleaned: string = pattern.toLowerCase().trim();
  return valueCleaned.includes(patternCleaned);
}
```

and then do:
`    (option) => contains(option.firstName, state.inputValue) || contains(options.email, state.inputValue)`

Reply:
Extracted the whole filtering function, as its own module. Looks good now at least to me.